src/components/EnterpriseApp/data/hooks/useUpdateActiveEnterpriseForUser.js

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect, useCallback } from 'react';
+import { logError } from '@edx/frontend-platform/logging';
+
+import LmsApiService from '../../../../data/services/LmsApiService';
+
+const useUpdateActiveEnterpriseForUser = ({
+  enterpriseId,
+  user,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+
+  const updateActiveEnterpriseAndRefreshJWT = useCallback(async (username, currentEnterpriseId) => {
```

Comment:
nit: likely don't necessarily need to pass `currentEnterpriseId`. This hook function itself gets `enterpriseId` as an argument, and it'd be available within `updateActiveEnterpriseAndRefreshJWT` without it as an explicit function argument.

Reply:
I didn't realize useCallback's get access to arguments passed to the hook!